examples/ui.apps/src/main/content/jcr_root/apps/cif-components-examples/clientlibs/venia-theme/venia.css

```diff
@@ -0,0 +1,3350 @@
+  --venia-error: 192, 18, 63;
+  --venia-error-alt: 255, 226, 234;
+  --venia-font: Muli, -apple-system, BlinkMacSystemFont, sans-serif;
+  --venia-grey: 246, 246, 246;
+  --venia-grey-dark: 209, 209, 209;
+  --venia-orange: 241, 99, 33;
+  --venia-teal: 0, 115, 120;
+  --venia-teal-alt: 224, 240, 241;
+  --venia-teal-dark: 0, 104, 108;
+  --venia-teal-light: 212, 243, 238;
+  --venia-text: 33, 33, 33;
+  --venia-text-alt: 112, 112, 112;
+  --venia-text-hint: 158, 158, 158;
+  --venia-text-spot: 255, 99, 51;
+  --venia-warning-dark: 249, 93, 94;
+  --venia-warning-light: 254, 229, 232;
+}
+.cif-demo {
+  /*
+ *  Copyright 2019 Adobe Systems Incorporated
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+  /*******************************************************************************
+ *
+ *    Copyright 2019 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+  /*******************************************************************************
+ *
+ *    Copyright 2019 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+  /*******************************************************************************
+ * ADOBE CONFIDENTIAL
```

Comment:
This should not be here.

Reply:
This file is generated: this header is coming from one of the `css` file, actually https://github.com/adobe/aem-cif-project-archetype/blob/master/src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appsFolderName__/clientlibs/theme/components/minicart/flow.css but there might be more with the "wrong" header. I didn't want to use any plugin to remove headers/comments or minimise the file: I'd rather invest more time to automate this in a separate issue.